packages/bot-skeleton/src/scratch/hooks/toolbox.js

```diff
@@ -75,14 +72,16 @@ Blockly.Toolbox.prototype.populate_ = function (newTree) {
 
     this.categoryMenu_.populate(newTree);
 
-    const { quick_strategy } = ScratchStore.instance;
-    const quick_strat_btn = document.createElement('BUTTON');
-    quick_strat_btn.innerHTML = localize('Quick Strategy');
-    quick_strat_btn.className = 'toolbox__button btn effect btn--primary btn__medium';
-    quick_strat_btn.id = 'gtm-quick-strategy';
-    quick_strat_btn.onclick = quick_strategy.toggleStrategyModal;
+    const { toggleStrategyModal } = DBotStore.instance;
+    if (toggleStrategyModal) {
```

Comment:
Same here.

Reply:
@easteregg there is no quick strategy in mobile.